src/sandbox/components/screens/call-functions/components/available-functions.tsx

```diff
@@ -0,0 +1,275 @@
+// @ts-nocheck
+import React from 'react';
```

Comment:
Unrelated to this code:

Nit: In the sandbox at a smaller height, the function list gets unnecessarily cut off. I only saw this because I had dev tools open.

![image](https://user-images.githubusercontent.com/1109058/108551191-73ec4880-72a4-11eb-838e-3431010e7a0e.png)